src/Playgrounds/CosmosEndToEnd/CosmosEndToEnd.ApiService/Program.cs

```diff
@@ -0,0 +1,70 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Text.Json.Serialization;
+using CosmosEndToEnd.ApiService;
+using Microsoft.Azure.Cosmos;
+
+var sessionId = Guid.NewGuid().ToString();
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+builder.AddAzureCosmosDB("db",
+                         settings =>
+                            {
+                                settings.IgnoreEmulatorCertificate = true;
+                            },
+                         clientOptions =>
+                         {
+                             // Default serializer for Cosmos V3 client is JSON.NET, this changes
+                             // us to use S.T.J for this playground.
+                             clientOptions.Serializer = new StjSerializer(new System.Text.Json.JsonSerializerOptions());
```

Comment:
Later follow-up: With next release of SDK there will be a new type for serialization to be implemented to handle LINQ queries.